test/link_transformer_test.rb

```diff
@@ -0,0 +1,87 @@
+require_relative "test_helper"
+
+describe LinkTransformer do 
+  let (:link_transformer) {LinkTransformer.new}
```

Comment:
* Style: No space between a method name and the opening parenthesis.
* Style: Put once space inside the curly braces when providing a block (e.g. `{ LinkTransformer.new }`)